docs/modules/ROOT/pages/backup-restore.adoc

```diff
@@ -168,8 +168,24 @@ include::ROOT:example$/hazelcast-persistence-restore-cr-name.yaml[]
 
 <1> `HotBackup` resource name used for restore. The backup folder name will be taken from the `HotBackup` resource.
 <2> Agent responsible for restoring data from the local storage.
-    The agent configuration is optional. If you give `restore` under `persistence` and do not pass the agent configuration, Hazelcast Platform Operator
-    uses the latest agent version that is compatible with its version.
+The agent configuration is optional. If you give `restore` under `persistence` and do not pass the agent configuration, Hazelcast Platform Operator
+uses the latest agent version that is compatible with its version.
+
+NOTE: For restore to be successful you have to give restore configuration at the `Hazelcast` custom resource creation. Otherwise, triggering restore on currently running members will likely to fail.
```

Comment:
```suggestion
NOTE: For restore to be successful you must configure restore when creating the `Hazelcast` custom resource. Otherwise, triggering restore on currently running members is likely to fail.
```